public/app/features/dashboard-scene/solo/SoloPanelPage.tsx

```diff
@@ -39,15 +40,26 @@ export function SoloPanelPage({ queryParams }: Props) {
     return <PageLoader />;
   }
 
-  return <SoloPanelRenderer dashboard={dashboard} panelId={queryParams.panelId} />;
+  return (
+    <UrlSyncContextProvider scene={dashboard}>
+      <SoloPanelRenderer dashboard={dashboard} panelId={queryParams.panelId} />
+    </UrlSyncContextProvider>
+  );
 }
 
 export default SoloPanelPage;
 
 export function SoloPanelRenderer({ dashboard, panelId }: { dashboard: DashboardScene; panelId: string }) {
   const [panel, error] = useSoloPanel(dashboard, panelId);
+  const { controls } = dashboard.useState();
+  const { refreshPicker } = controls!.useState();
```

Comment:
is there a way to avoid the need to use `!` here? This could result in a runtime error if `controls` is undefined for some reason 🤔

Reply:
why does the solo page need a refresh picker? I don't recall it ever having one, it's just the panel

Reply:
@torkelo It seems we need the scene object because the refresh picker is responsible for triggering the refresh and managing the interval for the next refresh. If the object isn't activated, the subsequent refresh won’t occur once the interval passes, and the URL parameter meant to set the refresh interval won’t function either.

Reply:
correct @ivanortegaalba . I don't know if there's any other way of activating the refresh, but I couldn't find anything in the panel context